inf-clojure.el

```diff
@@ -940,6 +940,25 @@ prefix argument PROMPT-FOR-SYMBOL, it prompts for a symbol name."
                (inf-clojure-symbol-at-point))))
     (comint-proc-query (inf-clojure-proc) (format (inf-clojure-var-source-form) var))))
 
+(defvar inf-clojure--dump-file-name ".inf-clojure.dump"
```

Comment:
Btw is there something like this in cider? Are we sure we want a buffer and not just a file? It is easier to implement (and maintain) I guess..